src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.SkipValue.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Cryptography.Encoding.Tests.Cbor
+{
+    internal partial class CborReader
+    {
+        public void SkipValue()
+        {
+            int depth = 0;
+
+            do
+            {
+                SkipNextNode(ref depth);
+
```

Comment:
Seems like the method could just be inlined here... does it need the separate method with the ref parameter?

If yes, does it need the extra blank line here? :smile:

Reply:
> does it need the separate method

I pulled an Uncle Bob here, it's only there to address readability.